src/Switch/Switch.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
```

Comment:
All the switches in storybook are acting weird, not super fussed about this as it is more about working in an app

Reply:
Its something weird about the state in the paeg I think. I don't know how to solve it

Reply:
This is fixed